overlays updates (#400)
mitre/saf-training: src/courses/beginner/10.md

In addition to its own controls, an InSpec profile can leverage controls from one or more other InSpec profiles.

> When a profile depends on controls from other profiles, it can be referred to as an *overlay profile*. We will call the profiles on which the overlay depends as the "baseline" profiles (there is no industry-standard term for the relationship here -- Chef, for instance, calls a baseline profile the "to-be-included" or the "child" profile in different parts of the InSpec docs).

Reviewer: ```suggestion
When a profile depends on controls from other profiles, it can be referred to as an *overlay profile*. We will call the profiles that the overlay depends on the “baseline” profiles (there is no industry-standard term for this relationship; for example, Chef calls a baseline profile a “to-be-included” profile or a “child” profile in different parts of the InSpec documentation)..
```

depends:
  - name: my_nginx
    path: ../my_nginx # {path relative to the overlay}
```
:::

> We give a relative path that includes shorthand for jumping up a directory here (`../my_nginx`) because the dependencies are processed relative to the root of the overlay profile (e.g. inside the `my_nginx_overlay` directory).

Reviewer: ```suggestion
We use a relative path that includes the shorthand `..` for jumping up a directory (`../my_nginx`) to jump up one level because dependencies are resolved relative to the root of the overlay profile (e.g., inside the `my_nginx_overlay` directory).
```
